File: app/src/main/java/com/castelijns/mmdemo/users/Address.java
package com.castelijns.mmdemo.users;

import java.util.Map;

public class Address {

    private String street;
    private String suite;
    private String city;
    private String zipcode;
    private Map<String, String> geo;

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public String getSuite() {
        return suite;
    }

    public void setSuite(String suite) {
        this.suite = suite;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getZipcode() {
        return zipcode;
    }

    public void setZipcode(String zipcode) {
        this.zipcode = zipcode;
    }

    public Map<String, String> getGeo() {
        return geo;
    }

    public void setGeo(Map<String, String> geo) {
        this.geo = geo;
    }
}

File: app/src/main/kotlin/com/castelijns/mmdemo/models/Address.kt
package com.castelijns.mmdemo.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Address(
        val street: String,
        val suite: String,
        val city: String,
        val zipcode: String,
        val geo: Geo
) : Parcelable

@Parcelize
data class Geo(
        val lat: String,
        val lng: String
) : Parcelable
